project/src/main/webapp/index.html

```diff
@@ -1,17 +1,25 @@
 <!DOCTYPE html>
 <html>
- <script src="https://polyfill.io/v3/polyfill.min.js?features=default"></script>
- <script src="https://maps.googleapis.com/maps/api/js?key=API_KEY&libraries=places"> </script>
- <script src="script.js"></script>
- <head>
-   <meta charset="UTF-8">
-   <title>food-mood</title>
-   <link rel="stylesheet" href="style.css">
-   <base target="_blank" rel="noopener noreferrer">
- </head>
- <body class="text-content">
-  <h1> FoodMood - your food order helper!</h1>
-  <div class="user-input" id="user-input">
+  <script src="https://polyfill.io/v3/polyfill.min.js?features=default"></script>
+  <script src="https://maps.googleapis.com/maps/api/js?key=API_KEY&libraries=places"> </script>
+  <script src="https://apis.google.com/js/platform.js"></script>
+  <meta name="google-signin-client_id" content="CLIENT_ID">
+  <script src="script.js"></script>
+  <head>
+    <meta charset="UTF-8">
+    <title>food-mood</title>
+    <link rel="stylesheet" href="style.css">
+    <base target="_blank" rel="noopener noreferrer">
+  </head>
+  <body class="text-content">
+    <h1> FoodMood - your food order helper!</h1>
+    <div id="sign-in-segment">
+      <div id="sign-in-button" class="g-signin2" data-onsuccess="onSignIn" data-prompt="select_account"></div>
+      <div id ="user-welcome-message-container"> Hello! You are welcome to sign in </div>
+      <button id="sign-out-button" onclick="signOut();">Sign out</button>
```

Comment:
Is the sign out button always displayed? Shouldn't you show it only if the user is signed out?

Reply:
Great idea! done